polars/polars-core/src/chunked_array/ops/unique/rank.rs

```diff
@@ -52,12 +56,65 @@ pub(crate) fn rank(s: &Series, method: RankMethod) -> Series {
             count += 1;
         });
     }
-    let inv_ca = UInt32Chunked::new_from_aligned_vec(s.name(), inv);
 
     use RankMethod::*;
     match method {
-        Ordinal => inv_ca.into_series(),
+        Ordinal => {
+            let inv_ca = UInt32Chunked::new_from_aligned_vec(s.name(), inv);
+            inv_ca.into_series()
+        }
+        Random => {
```

Comment:
Can we add a `[cfg(feature="random")]` on this branch? Then we can still use `rank` if we don't wnat to compile the `random` feature.

Reply:
added.